code/perception/src/radar_node.py

```
@@ -5,21 +5,102 @@
 from sklearn.cluster import DBSCAN
+from sklearn.cluster import HDBSCAN
 from sklearn.preprocessing import StandardScaler
 import json
 from sensor_msgs import point_cloud2
 from visualization_msgs.msg import Marker, MarkerArray
-from tf.transformations import quaternion_from_matrix
 from std_msgs.msg import Float32MultiArray
+from tf.transformations import quaternion_from_matrix
+
 import struct
+import numpy as np
```

Ruff (0.8.2)

Redefinition of unused `np` from line 4

Remove definition: `np`

(F811)